meetings/minutes/2022-09-12-TSC-Minutes.md

```diff
@@ -0,0 +1,93 @@
+# Minutes from the 2022-09-12 Laminas Technical Steering Committee Meeting
+
+- When: 2022-00-12 at 19:00 UTC, until 20:36 UTC
+- Where: Laminas Slack #tsc-meeting channel
+- Attending:
+  - Abdul Malik Ikhsan
+  - Aleksei Khudiakov
+  - Andreas Heigl (left after second agenda item)
+  - Frank
+  - Gary Lockett
+  - George Steel
+  - Luís Cobucci
+  - Marco Pivetta (left after second agenda item)
+  - Matthew Weier O'Phinney
+
+Quorum WAS met (9 out of 15 members were present).
+
+## Agenda
+
+### Default git branching for new majors
+
+When creating a new major, the relevant branch is created; it is also set as the default branch for the repository. However, there can be a significant gap between branch creation and the first release of that major version.
+
+Contributors and renovate, will attempt to make bugfixes to this new major when they equally apply to the previous (current latest release) major.
+
+I (ed: George Steel) propose that the default branch should only be changed to the new major branch, once a release has been tagged for that new major.
```

Comment:
just to point out to save @gsteel from any future hassle, this was my submission, I forgot to add my name in the original PR, my bad

Reply:
Updated!